### ISSUE TITLE
Markdown readme.md is not working

### ISSUE THREAD

**Issue by zoumingzhe** [adrianschubek/dir-browser#29]
**Describe the bug**

Create the following metadata (readme.md.dbmeta.json) for readme.md, the readme content on the page will not be displayed.

```readme.md.dbmeta.json
{
    "description": null,
    "labels": [],
    "hidden": true,
    "password": null,
    "hash_required": false
}
```

**Expected behavior**

When hidden is true, the readme.md content is also displayed.

**Environment**
- <ins>dir-browser version: 3.3.1      </ins>
 - OS: Windows
 - Browser: Chrome

**Comment by adrianschubek** [comment]
This is actually by design, I don't want to hide public files (e.g. readme.md is also visible in a GitHub repo file list). But I will consider adding a special file like "dbmeta.md" which will render just markdown and is hidden from the list.

**Comment by adrianschubek** [comment]
Implemented in v3.5 https://dir-demo.adriansoftware.de/examples/markdown/
